mwdb/resources/group.py

```diff
@@ -240,6 +240,56 @@ def put(self, name):
         schema = GroupSuccessResponseSchema()
         return schema.dump({"name": group.name})
 
+    @requires_authorization
+    @requires_capabilities(Capabilities.manage_users)
+    def delete(self, name):
+        """
+        ---
+        summary: Delete group
+        description: |
+            Remove group from database.
+
+            Requires `manage_users` capability.
+        security:
+            - bearerAuth: []
+        tags:
+            - group
+        parameters:
+            - in: path
+              name: name
+              schema:
+                type: string
+              description: Group name
+        responses:
+            200:
+                description: When group was removed successfully
+                content:
+                  application/json:
+                    schema: GroupSuccessResponseSchema
+            403:
+                description: When user doesn't have `manage_users` capability.
+            404:
+                description: When group doesn't exist
+        """
+        group = (
+            db.session.query(Group)
+            .options(joinedload(Group.members), joinedload(Group.members, Member.user))
```

Comment:
That joinedload seems to be not required because we don't operate on members :thinking:
```suggestion
```